flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why not aarch64?

Reply:
We currently have no upstream builds for aarch64.